roles/edpm_iscsid/handlers/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- name: Record the iscsid container restart is required
```

Comment:
Ah, good! The first time I looked at this PR, the handler was empty and I was wondering how it was all supposed to work! This makes sense now.